ui-tests/build.gradle

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+repositories {
+    maven { url = "https://jetbrains.bintray.com/intellij-third-party-dependencies" }
```

Comment:
oh no more bintray

Reply:
JetBrains puts everything on bintray. We are already using it, but transparently. (It is where JBR is downloaded from)